detection-rules/attachment_lnk_file.yml

```diff
@@ -0,0 +1,32 @@
+name: "Attachment: LNK file"
+description: |
+  Recursively scans files and archives to detect LNK connection files.
+
+  LNK files can be weaponised to execute arbitrary commands including unpacking and running executable content embedded within the file itself.
+references:
+  - "https://forensicitguy.github.io/shortcut-to-emotet-ttp-change/"
+type: "rule"
+authors:
+  - twitter: "ajpc500"
+source: |
+  type.inbound 
+  and (
+      any(attachments, .file_extension == "lnk")
```

Comment:
this makes sense to me. any `.lnk` are automatically included, and only archiver formats are recursed with binexplode. I like this pattern. I think it can be simplified a bit with a single `any(attachments`.

Is there any more logic that could be added? I think this rule is fine as a starting point, but it would probably need some additional logic to get less false positives

Reply:
>this makes sense to me.

i clearly had not had my morning coffee yet. thanks